2.0.0-rc1-7.3.0 build issue

Getting this

```
make build_image
./gradlew clean assemble

> Task :printGlobalBuildInfo UP-TO-DATE
=======================================
Elasticsearch Build Hamster says Hello!
  Gradle Version        : 5.5
  OS Info               : Linux 5.0.0-29-generic (amd64)
  JDK Version           : 12 (Private Build 12.0.2 [OpenJDK 64-Bit Server VM 12.0.2+9-Ubuntu-119.04])
  JAVA_HOME             : /usr/lib/jvm/java-12-openjdk-amd64
  Random Testing Seed   : F2E58CFA044E78A8
=======================================

> Task :compileJava
Note: /home/username/code/es-lattice-index-plugin/src/main/java/com/eigendomain/eslatticeindex/index/query/LatticePayloadScoreQuery.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

BUILD SUCCESSFUL in 2s
12 actionable tasks: 11 executed, 1 up-to-date
docker build --no-cache -t "messiaen/elasticsearch-plug-ph-lat":2.0.0-rc1 .
Sending build context to Docker daemon  11.74MB
Step 1/4 : FROM docker.elastic.co/elasticsearch/elasticsearch:7.3.0
 ---> bdaab402b220
Step 2/4 : COPY build/distributions/es-lattice-index-plugin-2.0.0-7.3.0.zip /es-lattice-index-plugin.zip
COPY failed: stat /var/lib/docker/tmp/docker-builder842084929/build/distributions/es-lattice-index-plugin-2.0.0-7.3.0.zip: no such file or directory
make: *** [Makefile:13: build_image] Error 1
```

Let me know if you want to use ARG for the version.  Simple fix is to update the Dockerfile

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,7 +1,9 @@
 FROM docker.elastic.co/elasticsearch/elasticsearch:7.3.0
 
+ARG VERSION=v1.0.0
+
 # version should match VERSION.txt plus hyphen plus ES version from gradle.properties
-COPY build/distributions/full-lattice-search-2.0.0-rc2-7.3.0.zip /full-lattice-search.zip
+COPY build/distributions/full-lattice-search-${VERSION}.zip /full-lattice-search.zip
 
 RUN bin/elasticsearch-plugin install analysis-phonetic
 RUN bin/elasticsearch-plugin install file:///full-lattice-search.zip && rm -f /full-lattice-search.zip

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,5 +1,6 @@
 FROM docker.elastic.co/elasticsearch/elasticsearch:7.3.0
 
+# version should match VERSION.txt plus hyphen plus ES version from gradle.properties
 COPY build/distributions/full-lattice-search-2.0.0-rc2-7.3.0.zip /full-lattice-search.zip
 
 RUN bin/elasticsearch-plugin install analysis-phonetic

--- a/Makefile
+++ b/Makefile
@@ -1,12 +1,16 @@
+# Disable built-in rules and variables
+MAKEFLAGS += --no-builtin-rules
+MAKEFLAGS += --no-builtin-variables
+
 ES_VERSION = $(shell grep elasticsearchVersion gradle.properties  | cut -d " " -f 3)
 VERSION = $(shell cat VERSION.txt)
 NAME = "registry.gitlab.com/hedgehogai/full-lattice-search/full-lattice-search"
 IMAGE_VERSION = $(VERSION)-$(ES_VERSION)
 
+default: build_plugin
+
 clean:
 	./gradlew clean
-
-default: build_plugin
 
 build_plugin:
 	./gradlew clean assemble

--- a/Makefile
+++ b/Makefile
@@ -16,7 +16,7 @@ build_plugin:
 	./gradlew clean assemble
 
 build_image: build_plugin
-	docker build --no-cache -t $(NAME):$(IMAGE_VERSION) .
+	docker build --no-cache --build-arg VERSION=$(IMAGE_VERSION) -t $(NAME):$(IMAGE_VERSION) .
 	docker tag $(NAME):$(IMAGE_VERSION) $(NAME):latest
 
 push:

--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,7 @@
+ES_VERSION = $(shell grep elasticsearchVersion gradle.properties  | cut -d " " -f 3)
 VERSION = $(shell cat VERSION.txt)
 NAME = "registry.gitlab.com/hedgehogai/full-lattice-search/full-lattice-search"
+IMAGE_VERSION = $(VERSION)-$(ES_VERSION)
 
 clean:
 	./gradlew clean
@@ -10,11 +12,11 @@ build_plugin:
 	./gradlew clean assemble
 
 build_image: build_plugin
-	docker build --no-cache -t $(NAME):$(VERSION) .
-	docker tag $(NAME):$(VERSION) $(NAME):latest
+	docker build --no-cache -t $(NAME):$(IMAGE_VERSION) .
+	docker tag $(NAME):$(IMAGE_VERSION) $(NAME):latest
 
 push:
-	docker push $(NAME):$(VERSION)
+	docker push $(NAME):$(IMAGE_VERSION)
 	docker push $(NAME):latest
 
 test:

